dockers/docker-fpm-quagga/start.sh

```diff
@@ -1,7 +1,11 @@
 #!/usr/bin/env bash
 
 mkdir -p /etc/quagga
-sonic-cfggen -m /etc/sonic/minigraph.xml -t /usr/share/sonic/templates/bgpd.conf.j2 > /etc/quagga/bgpd.conf
+if [ -f /etc/sonic/bgp_admin.yml ]; then
+    sonic-cfggen -m /etc/sonic/minigraph.xml -y /etc/sonic/bgp_admin.yml -t /usr/share/sonic/templates/bgpd.conf.j2 > /etc/quagga/bgpd.conf
```

Comment:
a sample file of bgp_admin.yml file could be provided in sonic-cfggen tests folder.